build.js

```diff
@@ -11,6 +11,20 @@ const includeFiles = [
   'spinner.css', 'supported_extensions.js',
 ];
 const zipName = './gecko-code-coverage.zip';
+fs.readFile('manifest.json', 'utf8', function(err, data) {
+  var manObj;
```

Comment:
You don't need to define the variable here, you can directly define it when you need it (`var manObj = JSON.parse(data);`).

Also use `let` instead of `var`.